### ISSUE TITLE
[BUG] Interpreter seems not to be able to handle substructured field properly

### ISSUE THREAD

**Issue by Shinyaigeek** [ysugimoto/falco#177]
Thank you for developing such a nice tool!! and congratulations for the 1.0.0 rc release 🎉 I tried to apply `falco@1.0.0-rc1`'s local testing function to my product, and I found this issue.

**Describe the problem**

(Maybe this post includes two issues 🙏 )

falco's interpreter seems not to be able to handle substructured field properly in HTTP Header Field, both of setter/getter.

**VCL code that cause the problem**

```vcl
sub vcl_recv {
    set req.http.VARS:hoge = "hoge";
    set req.http.VARS:fuga = "fuga";
    // @debugger
    set req.http.VARS:bar = "bar";
}
```

also, I make the small reproduction https://github.com/Shinyaigeek/falco-interpreter-playground

I executed

```shell
./falco simulate --debug ./default.vcl
```

and launched the simulator's debugger and checked `req.http.VARS` and `req.http.VARS:hoge`

**Screenshots**

if I call `req.http.VARS`, falco outputs the below result

```
│> req.http.VARS                                                                              │
│>> (STRING)hoge=hoge 
```

if I call `req.http.VARS:hoge`, falco outputs the below result

```
│> req.http.VARS:hoge                                                                         │
│>> (STRING)
```

**Expected behavior**

In calling `req.http.VARS`, the expected outputs are the below

```
│> req.http.VARS                                                                              │
│>> (STRING)hoge=hoge; fuga=fuga
```

In calling `req.http.VARS:hoge`, the expected outputs are the below

```
│> req.http.VARS:hoge                                                                         │
│>> (STRING) hoge
```

**Desktop (please complete the following information):**
 - OS: macos
 - Shell: zsh

**Comment by ysugimoto** [comment]
@Shinyaigeek Thanks for finding an issue with more details!
It seems a bug, I will look into it.

**Comment by ysugimoto** [comment]
This issue should be fixed in `1.0.0-rc2` including #178 , please try the new RC version.
In addition,  multiple headers like `VARS:xxx`  will be combined with a comma:

```
set req.http.VARS:foo = "bar";
set req.http.VARS:hoge = "huga";

On HTTP request:
vars: foo=bar,hoge=hufga
```

You can see the above behavior in [fiddle](https://fiddle.fastly.dev/fiddle/15a5fccd)

Thank you for reporting the issue.

**Comment by Shinyaigeek** [comment]
@ysugimoto This worked perfectly on my environment!! Thank you for the quick fix 👍 (and sorry for mistake 🤦)
